.github/workflows/build-plugin-archive.yml

```
@@ -0,0 +1,183 @@
+name: Create plugin archive
+
+on:
+  workflow_call:
+    inputs:
+      PLUGIN_MAIN_FILE:
+        description: The name of the main plugin file.
+        required: false
+        default: 'index.php'
+        type: string
+      PLUGIN_VERSION:
+        description: The new plugin version.
+        required: true
+        type: string
+      ARCHIVE_NAME:
+        description: The base name of the resulting zip archive. Falls back to the repository name.
+        default: ''
+        required: false
+        type: string
+      COMPOSER_ARGS:
+        description: Set of arguments passed to Composer for production setup.
```

Now that we no longer have `COMPOSER_ARGS_DEV`, please adjust this description accordingly. 
```suggestion
        description: Set of arguments passed to Composer.
```